### PR TITLE
[#IOPAE-1137] Add GetFeaturedServices and GetFeaturedInstitutions fn

### PR DESCRIPTION
**Pull request by StefanoDoc**
Change `/featured` API into 2 new api `/institutions/featured` and `/services/featured`
#### List of Changes

- add new GET `/institutions/featured`
- add new GET `/services/featured`
- remove old GET `/featured`
- Update config and infra

#### How Has This Been Tested?
unit tests

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by giuseppedipinto — CHANGES_REQUESTED** [review]

File: infra/prod/italynorth/main.tf [hunk 1]
```diff
@@ -58,8 +58,9 @@ module "function_app" {
   }
 
   app_be_fn_settings = {
-    FEATURED_ITEMS_CONTAINER_NAME = "static-content"
-    FEATURED_ITEMS_FILE_NAME      = "featured-items.json"
+    FEATURED_ITEMS_CONTAINER_NAME         = "static-content"
+    FEATURED_SERVICES_ITEMS_FILE_NAME     = "featured-services-items.json"
+    FEATURED_INSTITUTIONS_ITEMS_FILE_NAME = "featured-institutions-items.json"
```
[the remark this answers is not in the document]
Comment (StefanoDoc) [reply infra/prod/italynorth/main.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: infra/prod/italynorth/main.tf [hunk 2]
```diff
@@ -58,8 +58,9 @@ module "function_app" {
   }
 
   app_be_fn_settings = {
-    FEATURED_ITEMS_CONTAINER_NAME = "static-content"
-    FEATURED_ITEMS_FILE_NAME      = "featured-items.json"
+    FEATURED_ITEMS_CONTAINER_NAME         = "static-content"
+    FEATURED_SERVICES_ITEMS_FILE_NAME     = "featured-services-items.json"
```
[the remark this answers is not in the document]
Comment (StefanoDoc) [reply infra/prod/italynorth/main.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: infra/_modules/function_app/variables.tf [hunk 3]
```diff
@@ -101,8 +101,9 @@ variable "fn_zone_balancing_enabled" {
 
 variable "app_be_fn_settings" {
   type = object({
-    FEATURED_ITEMS_CONTAINER_NAME = string
-    FEATURED_ITEMS_FILE_NAME      = string
+    FEATURED_ITEMS_CONTAINER_NAME         = string
+    FEATURED_SERVICES_ITEMS_FILE_NAME     = string
+    FEATURED_INSTITUTIONS_ITEMS_FILE_NAME = string
```
[the remark this answers is not in the document]
Comment (StefanoDoc) [reply infra/_modules/function_app/variables.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: infra/_modules/function_app/variables.tf [hunk 4]
```diff
@@ -101,8 +101,9 @@ variable "fn_zone_balancing_enabled" {
 
 variable "app_be_fn_settings" {
   type = object({
-    FEATURED_ITEMS_CONTAINER_NAME = string
-    FEATURED_ITEMS_FILE_NAME      = string
+    FEATURED_ITEMS_CONTAINER_NAME         = string
+    FEATURED_SERVICES_ITEMS_FILE_NAME     = string
```
[the remark this answers is not in the document]
Comment (StefanoDoc) [reply infra/_modules/function_app/variables.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: apps/app-backend/src/config.ts [hunk 5]
```diff
@@ -24,7 +24,8 @@ export const FeaturedItemsConfig = t.type({
   // TODO: Capire se metterlo direttamente nello storageAccount della function in tal caso recuperarlo come QueueStorageConnection nella sezione globale
   FEATURED_ITEMS_BLOB_CONNECTION_STRING: NonEmptyString,
   FEATURED_ITEMS_CONTAINER_NAME: NonEmptyString,
-  FEATURED_ITEMS_FILE_NAME: NonEmptyString,
+  FEATURED_SERVICES_ITEMS_FILE_NAME: NonEmptyString,
+  FEATURED_INSTITUTIONS_ITEMS_FILE_NAME: NonEmptyString,
```
[the remark this answers is not in the document]
Comment (StefanoDoc) [reply apps/app-backend/src/config.ts]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: apps/app-backend/src/config.ts [hunk 6]
```diff
@@ -24,7 +24,8 @@ export const FeaturedItemsConfig = t.type({
   // TODO: Capire se metterlo direttamente nello storageAccount della function in tal caso recuperarlo come QueueStorageConnection nella sezione globale
   FEATURED_ITEMS_BLOB_CONNECTION_STRING: NonEmptyString,
   FEATURED_ITEMS_CONTAINER_NAME: NonEmptyString,
-  FEATURED_ITEMS_FILE_NAME: NonEmptyString,
+  FEATURED_SERVICES_ITEMS_FILE_NAME: NonEmptyString,
```
[the remark this answers is not in the document]
Comment (StefanoDoc) [reply apps/app-backend/src/config.ts]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

**Review by giamma1295 — COMMENTED** [review]
LGTM but wait for checks on infra itn action failure before merge!

**Review by giuseppedipinto — APPROVED** [review]

**Pull request merged**